week-1/k8s_config/application.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: sample-pod
+spec:
+  containers:
+  - name: sample-app
+    image: sample:v1
```

Comment:
should we pull from the registry instead locally?

![image](https://github.com/user-attachments/assets/135a4818-eaa2-4d2f-bdd5-a07cf61532dd)